src/yunohost/tools.py

```diff
@@ -982,208 +1019,206 @@ def tools_migrations_list(pending=False, done=False):
     # Get all migrations
     migrations = _get_migrations_list()
 
-    # If asked, filter pending or done migrations
-    if pending or done:
-        last_migration = tools_migrations_state()["last_run_migration"]
-        last_migration = last_migration["number"] if last_migration else -1
-        if done:
-            migrations = [m for m in migrations if m.number <= last_migration]
-        if pending:
-            migrations = [m for m in migrations if m.number > last_migration]
-
     # Reduce to dictionnaries
     migrations = [{"id": migration.id,
                    "number": migration.number,
                    "name": migration.name,
                    "mode": migration.mode,
+                   "state": migration.state,
                    "description": migration.description,
                    "disclaimer": migration.disclaimer} for migration in migrations]
 
+    # If asked, filter pending or done migrations
+    if pending or done:
+        if done:
+            migrations = [m for m in migrations if m["state"] != "pending"]
```

Comment:
```suggestion
            migrations = [m for m in migrations if m["state"] != "done"]
```

?

Reply:
like  that --done return  "done" and "skip" but i think webadmin want to display done and skipped migrations together.

Reply:
Marking as resolved (c.f. ljf's comment)